test/parallel_api/numeric/numeric.ops/histogram.pass.cpp

```diff
@@ -107,20 +124,21 @@ test_histogram(T min_boundary, T max_boundary, T overflow, Size jitter, Size tra
             test_range_and_even_histogram<CallNumber>(n, min_boundary, max_boundary, overflow, jitter, bin_size, trash);
         }
     }
+
+    //single large case for coverage
+    Size n = 1100000;
+    Size bin_size = 1100000;
```

Comment:
```suggestion
    const Size n = 1100000;
    const Size bin_size = 1100000;
```

Reply:
This case actually isnt necessary for coverage anymore without the atomic version of the algorithm, so I've removed it.